<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_Bshelf">
        <property name="title">
         <string>ШКАФЫ</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QListWidget" name="listWidget_Bshelf"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButton_add_Bshelf">
             <property name="text">
              <string>ДОБАВИТЬ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_dell_Bshelf">
             <property name="text">
              <string>УДАЛИТЬ</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_shelf">
        <property name="title">
         <string>ПОЛКИ</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QListWidget" name="listWidget_shelf"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="pushButton_add_shelf">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>ДОБАВИТЬ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_dell_shelf">
             <property name="text">
              <string>УДАЛИТЬ</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_book">
        <property name="title">
         <string>КНИГИ</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QListWidget" name="listWidget_book"/>
         </item>
         <item>
          <widget class="QLabel" name="label_author">
           <property name="text">
            <string>АВТОР</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_author"/>
         </item>
         <item>
          <widget class="QLabel" name="label_name">
           <property name="text">
            <string>НАЗВАНИЕ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_name"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_pages">
             <property name="text">
              <string>СТРАНИЦЫ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_pages">
             <property name="maximum">
              <number>50000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="pushButton_add_book">
             <property name="text">
              <string>ДОБАВИТЬ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_dell_book">
             <property name="text">
              <string>УДАЛИТЬ</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
